packages/@tailwindcss-upgrade/src/utils/extract-static-plugins.ts

```diff
@@ -5,66 +5,137 @@ let parser = new Parser()
-        (pair
-          key: (property_identifier) @_name (#eq? @_name "plugins")
-          value: (array) @imports
-        )
-      ))?
-      value: (as_expression (object
-        (pair
-          key: (property_identifier) @_name (#eq? @_name "plugins")
-          value: (array) @imports
-        )
-      ))?
-      value: (object
-        (pair
-          key: (property_identifier) @_name (#eq? @_name "plugins")
-          value: (array) @imports
-        )
-      )?
-    )
-
-    ; module.exports = {}
-    (expression_statement
-      (assignment_expression
-        left: (member_expression) @left (#eq? @left "module.exports")
-        right: (satisfies_expression (object
+      value: [
+        (satisfies_expression (object
           (pair
             key: (property_identifier) @_name (#eq? @_name "plugins")
             value: (array) @imports
           )
-        ))?
-        right: (as_expression (object
+        ))
+        value: (as_expression (object
           (pair
             key: (property_identifier) @_name (#eq? @_name "plugins")
             value: (array) @imports
           )
-        ))?
-        right: (object
+        ))
+        value: (object
           (pair
             key: (property_identifier) @_name (#eq? @_name "plugins")
             value: (array) @imports
           )
-        )?
+        )
+      ]
+    )
+
+    ; module.exports = {}
+    (expression_statement
+      (assignment_expression
+        left: (member_expression) @left (#eq? @left "module.exports")
+        right: [
+          (satisfies_expression (object
+            (pair
+              key: (property_identifier) @_name (#eq? @_name "plugins")
+              value: (array) @imports
+            )
+          ))
+          (as_expression (object
+            (pair
+              key: (property_identifier) @_name (#eq? @_name "plugins")
+              value: (array) @imports
+            )
+          ))
+          (object
+            (pair
+              key: (property_identifier) @_name (#eq? @_name "plugins")
+              value: (array) @imports
+            )
+          )
+        ]
+      )
+    )
+  `,
+)
+
+// Extract require() calls, as well as identifiers with options or require()
+// with options
+const PLUGIN_CALL_OPTIONS_QUERY = new Parser.Query(
+  TS.typescript,
+  treesitter`
+    (call_expression
+      function: [
+        (call_expression
+          function: (identifier) @_name (#eq? @_name "require")
+          arguments: (arguments
+            (string (string_fragment) @module_string)
+          )
+        )
+        (identifier) @module_identifier
+      ]
+      arguments: (arguments
+        (object
+          (pair
+            key: [
+              (property_identifier) @property
+              (string (string_fragment) @property)
+            ]
+
+            value: [
+              (string (string_fragment) @str_value)
+              (template_string (string_fragment) @str_value)
```

Comment:
Do you know if this would this pick up tagged template literals? If it does we should make sure they prevent migration.